doc/user/content/transform-data/patterns/temporal-filters.md

```diff
@@ -9,25 +9,48 @@ menu:
     parent: 'sql-patterns'
 ---
 
-A _temporal filter_ is a `WHERE` or `HAVING` clause which uses the [`mz_now()`](/sql/functions/now_and_mz_now) function.
-This function returns Materialize's current virtual timestamp, which works to keep up with real time as data is processed.
-Applying a temporal filter reduces the working dataset, saving memory resources and focusing results on the recent past.
-
-Here is a typical temporal filter that considers records whose timestamps are within the last 5 minutes.
+A **temporal filter** is a query condition/predicate that filters data based on
+a time-related (i.e., temporal) condition. Using a temporal filter reduces the
+working dataset, saving memory resources and focusing on results that are within
+the timeframe specified by the filter.
+
+In Materialize, you implement temporal filters using the
+[`mz_now()`](/sql/functions/now_and_mz_now) function (which returns
+Materialize's current virtual timestamp and keeps up with real time) in a
+`WHERE` or `HAVING` clause; specifically, you compare
+[`mz_now()`](/sql/functions/now_and_mz_now) to a numeric or timestamp column
+expression. As [`mz_now()`](/sql/functions/now_and_mz_now) progresses (every
+millisecond), records that fall outside the timeframe are retracted from the
+working dataset. That is, when using temporal filters, Materialize maintains (in
```

Comment:
For me, the "That is" doesn't check out in that this sentence does not restate the previous sentence. I think it is also not necessarily correct, especially in light of @antiguru's recent work (which .. makes it not correct). Here's a different framing:

> When using temporal filters Materialize must be prepared to retract updates in the near future, and will need resources to maintain these retractions. Temporal filter retractions in the far future may not need to be retained, depending on how you have configured your replica expiration: retractions that will occur after a replica expires can be discarded, and will not require resources to maintain.

We'll want to either track down or create some docs about replica expiration.

Reply:
Will update to your framing.  As for replica explication -- if I can track down, can add a link.  Otherwise, will add to my tasks worksheet as a todo for Jan.

Reply:
Ah ... nothing as of yet on replica expiration. Will save the second sentence for when we doc the expiration.